app/usecase/create_account.go

```diff
@@ -0,0 +1,60 @@
+package usecase
+
+//packages imports.
+import (
+	"errors"
+
+	"github.com/google/uuid"
+	"github.com/jirlon/digitalbank/app/entities"
+	"github.com/jirlon/digitalbank/app/repository"
+)
+
+// Defines the interface for the account creation use case.
+type CreateAccountUseCase interface {
+	CreateAccount(id, cpf, name, secret string) (*entities.Account, error)
+}
```

Comment:
Essa interface aqui não vai ser utilizada pelo domínio, então podemos remover
```suggestion
// Defines the interface for the account creation use case.
```